syntax = "proto3";

option java_package = "sample.grpc";

package sample;

import "google/protobuf/empty.proto";

service SampleService {
    rpc SampleStreamMethod (SampleRequest) returns (stream SampleResponse) {}
}

message SampleRequest {
    string request_sample_field = 1;
}

message SampleField {
    string response_sample_field = 1;
}

message SampleResponse {
    oneof oneof_sample{
        SampleField sample_field = 1;
        google.protobuf.Empty ping = 2;
    }
}
